omc3/model/model_creators/lhc_model_creator.py

```diff
@@ -82,6 +85,13 @@ def get_correction_check_script(
 
     @classmethod
     def prepare_run(cls, accel: Lhc) -> None:
+        print(f"year: {accel.year}")
+        if accel.year in ["2018", "2022"]:  # this should be handled by the fetcher
+            symlink_dst = Path(f"{accel.model_dir}/acc-models-lhc")
+            print(f"symlink dst: {symlink_dst}")
+            if symlink_dst.is_symlink():
+                os.unlink(symlink_dst)
+            os.symlink(f"{ACCMDLS}/{accel.year}", symlink_dst.absolute())
```

Comment:
why os when pathlib can do the same?

Reply:
why pathlib when os can do the same?

Reply:
because it was used in this script before, so in order to keep consistent, avoid extra imports, and any potential future incompatibilities between those modules .. who is to say that the unlink still works in 3 years with a path provided